common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import, print_function
+
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from os import remove
+
+from djqscsv import write_csv
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command will export a csv of all users who have logged in within the last six months and
+    have staff access role in active courses (Courses with end date in the future).
+    """
+
+    def handle(self, *args, **options):
+        current_date = datetime.now()
+        last_week_date = current_date - timedelta(days=7)
```

Comment:
We can maybe add the time duration as the command argument

Reply:
@asadazam93 ^^

Reply:
Once you add days as command-line arg, have a method to calculate the date here.